x-pack/test_serverless/api_integration/test_suites/observability/config.ts

```diff
@@ -21,6 +21,10 @@ export default createTestConfig({
   // include settings from project controller
   // https://github.com/elastic/project-controller/blob/main/internal/project/observability/config/elasticsearch.yml
   esServerArgs: ['xpack.ml.dfa.enabled=false'],
-  // defined in MKI control plane
-  kbnServerArgs: ['--xpack.uptime.service.manifestUrl=mockDevUrl'],
+  kbnServerArgs: [
+    // defined in MKI control plane
+    '--xpack.uptime.service.manifestUrl=mockDevUrl',
+    // useful for testing (also enabled in MKI QA)
+    '--coreApp.allowDynamicConfigOverrides=true',
```

Comment:
We need to be explicit because it is now disabled by default in the Serverless build (https://github.com/elastic/kibana/pull/184476)